test/e2e/specs/nux.test.js

```diff
@@ -142,4 +142,34 @@ describe( 'New User Experience (NUX)', () => {
 		expect( nuxTipsLocalStorage.areTipsEnabled ).toEqual( true );
 		expect( Object.keys( nuxTipsLocalStorage.dismissedTips ) ).toHaveLength( 0 );
 	} );
+
+	// TODO: This test should be enabled once
+	// https://github.com/WordPress/gutenberg/issues/7753 is fixed.
+	// See: https://github.com/WordPress/gutenberg/issues/7753#issuecomment-403952816
+	it.skip( 'should show tips if "Show tips" was disabled on a draft and then enabled', async () => {
+		// Disable tips.
+		await clickOnMoreMenuItem( 'Show Tips' );
```

Comment:
The fact that you click on `Show Tips` to disable is confusing at first glance, but it is what it is.

Reply:
Oh, good point. I'll add a comment clarifying that.